pkg/models/html.go

```diff
@@ -0,0 +1,39 @@
+package models
+
+import (
+	htmlPackage "html"
+	"html/template"
+
+	"github.com/cmsgov/mint-app/pkg/sanitization"
+)
+
+// hTML represents html code. It is sanitized when unmarshaled from graphQL or when converted to hTML to only allow specific tags
+type hTML string
```

Comment:
By not exporting the type, we can effectively control when the data is being sanitized.

Reply:
I think per go standards this might actually just be `html` instead of `hTML`

Reply:
thanks Clay, I updated the name.